src/VTMap.js

```diff
@@ -0,0 +1,371 @@
+import React from 'react';
+import './App.css';
+import { Map, TileLayer, Polygon, Polyline } from 'react-leaflet';
+import borderData from './border.js';
+import L from 'leaflet'
+import leafletPip from 'leaflet-pip'
+let defaultZoom = 7
+
+class VTMap extends React.Component {
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            latitude: '',
+            longitude: '',
+            county: '',
+            town: '',
+            status: 'Game Started, Awaiting your first guess..',
+            guess: '',
+            modalDisplayed: false,
+            points: 100,
+            mapLat: 44.0886,
+            mapLon: -72.7317,
+            locationArray: [],
+            highScore: '',
+            username: 'Unknown'
+            
+        }
+    }
+
+    // itterate over indexs [0]  and [1] in each array in the list of arrays and return the highest of each value
+    getCorners = () => {
+
+        let maxLon = -73.35218
+        let minLon = -71.510225
+
+        let maxLat = 45.005419
+        let minLat = 42.730315
+        return (
+            [minLat, maxLat, minLon, maxLon]
+        )
+    }
+
+    getRndInteger = (min, max) => {
+        return Math.floor(Math.random() * (max - min)) + min;
+    }
```

Comment:
This is completely breaking your game.  100%.  There's nothing random about it when it always rounds down!!!!

Reply:
There are two possible points your game drops the player at.  Lose the `Math.floor`!